python/pyspark/sql/functions.py

```diff
@@ -994,12 +1014,22 @@ def cot(col: "ColumnOrName") -> Column:
     Parameters
     ----------
     col : :class:`~pyspark.sql.Column` or str
-        Angle in radians
+        angle in radians.
```

Comment:
Why these changes? seemed better before

Reply:
There are different conventions in this file and most prominent ones is lower case and dot at the end. Tried to be consistent. Can revert if needed